app/Controllers/importExportController.php

```diff
@@ -64,31 +64,24 @@ public function importFile($name, $path, $username = null) {
 
 		// We try to list all files according to their type
 		$list = array();
-		if ($type_file === 'zip' && extension_loaded('zip')) {
-			$zip = zip_open($path);
-			if (!is_resource($zip)) {
+		if ('zip' === $type_file && extension_loaded('zip')) {
+			$zip = new ZipArchive();
+			$result = $zip->open($path, ZipArchive::RDONLY);
```

Comment:
The constant `ZipArchive::RDONLY` is first available from PHP 7.4.3 https://php.net/zip.constants

Reply:
Thanks, I haven't pay attention to that. We might as well drop it.